.github/workflows/ci.yml

```
@@ -180,24 +180,26 @@ jobs:
           sudo usermod -a -G audio "${USER}"
           sudo bash ci/setup-xvfb.sh
 
-      - name: Set up snd-dummy
+      - name: Check if linux-modules-extra is available
+        id: check-snd-dummy
         if: (!(contains(matrix.extra, 'unittests') || contains(matrix.extra, 'vimtags')))
```

You can try this condition:

```suggestion
        if: (!(contains(matrix.extra, 'unittests') || contains(matrix.extra, 'vimtags'))) && runner.name != 'ubuntu-22.04'
```

[runner context](https://docs.github.com/en/actions/learn-github-actions/contexts#runner-context)